roles/cfg_openwrt/templates/corerouter/config/dhcp.j2

```diff
@@ -27,6 +27,7 @@ config dhcp 'dhcp_{{ name }}'
 	option leasetime '5m'
 	option start '2'
 	option limit '{{ network['prefix'] | ipaddr('size') -3 }}'
+	list dhcp_option '119,olsr'
```

Comment:
This is not related to segen bringup? Please open a seperate PR, also explaining what would be the advantage with that change